console-csharp-connect-sample/Constants.cs

```diff
@@ -1,18 +1,14 @@
 ﻿//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
 //See LICENSE in the project root for license information.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 namespace console_csharp_connect_sample
 {
     internal class Constants
     {
-        public const string ClientId = "ENTER_YOUR_CLIENT_ID";
+		public const string ClientId = ""; // "ENTER_YOUR_CLIENT_ID";
```

Comment:
Typically we put these in config files and not in compiled code. @jasonjoh  can share some best practices here.

Reply:
It's best to move this to a dedicated file, possibly a .config file here. The idea is that you NEVER want to check this file in since it contains your app ID (and possibly tenant ID). So you would add that file to .gitignore. Create a copy of the file (with the extension .example), and insert placeholders. Then in your README, tell devs to rename that file and plug in their own values.